### PR TITLE
Add 0.20.x Knative manifests

### PR DESCRIPTION
**Pull request by houshengbo**
I just opened this PR for testing.
Have no idea why automated PR is still not opened.

### CODE REVIEW

File: test/e2e-eventing-upgrade-tests.sh
Comment (evankanderson) [inline test/e2e-eventing-upgrade-tests.sh:86]: Add a comment why these are commented out?

**Review by evankanderson — COMMENTED** [review]
/lgtm

/hold

In case you need to put a comment in for the tests -- a lot of the steps seem to have gotten commented out, but I didn't see the reason in the PR

**Review by maximilien — APPROVED** [review]
First time taking a look at code and PRs. Guessing this is all recurring changes for next update. Agree also with @evankanderson in document commented code or removing lines :)

/ok-to-test since something is failing

**Comment by houshengbo** [comment]
https://github.com/knative/operator/issues/428
I opened an issue tracking the framework transition work for upgrade.
We are targeting 0.21 to resolve them.
By far, we won't block the coming 0.20 release of operator.
So I tentatively disable pull-knative-operator-serving-upgrade-tests and pull-knative-operator-eventing-upgrade-tests.
Will work on fixing the pull-knative-operator-upgrade-tests.

File: test/e2e-eventing-upgrade-tests.sh
Comment (houshengbo) [reply test/e2e-eventing-upgrade-tests.sh:86]: I need to fix them, since the upgrade framework has changed in both serving and eventing repos.
I want to temporarily disable the upgrade tests here, targeting to resolve them in 0.21.
Not to block the coming 0.20 release.
Once this is fix in 0.21 or master, I will seek for backport them to 0.20 branch.

**Comment by houshengbo** [comment]
@evankanderson I added the comment.
As I described, we will resolve the prow issues asap in 0.21, without blocking the 0.20 release.
Once this PR is merged, we are ready to ship 0.20.0.

**Review by evankanderson — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**
